api/src/convert/validated_mint_config.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Convert to/from
+//! external:ValidatedMintConfigTx/ValidatedMintConfigTxTxPrefix/
+//! ValidatedMintConfigTxTx.
```

Comment:
```suggestion
//! Convert between the Rust and Protobuf versions of [ValidatedMintConfigTx]
```
(square brackets result in a link in docs)